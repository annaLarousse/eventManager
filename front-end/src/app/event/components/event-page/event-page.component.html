<div class="event-page__header">
  <img class="logo" src="../../../../assets/logo.svg" alt="Event Manager" />
</div>

<div class="event-page__container">
  <app-event-add></app-event-add>

  <mat-divider></mat-divider>

  <app-event-list></app-event-list>
</div>
