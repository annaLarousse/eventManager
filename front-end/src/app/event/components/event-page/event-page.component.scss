:host.event-page {
  .event-page {
    &__header {
      background-color: var(--primary);
      .logo {
        margin: 2rem;
        width: 22rem;
        animation: intro 3s ease-in-out;
      }
    }
    &__container {
      margin: 2rem;

      .mat-divider {
        margin: 2rem 0;
        border-top-color: var(--primary);
      }
    }
  }
}

@keyframes intro {
  from {
    opacity: 0;
  }
  to {
    opacity: 1;
  }
}
