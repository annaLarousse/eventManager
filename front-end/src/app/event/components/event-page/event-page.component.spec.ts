import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
import { ComponentFixture, TestBed } from '@angular/core/testing';

import { EventPageComponent } from './event-page.component';

describe('EventPageComponent', () => {
  let component: EventPageComponent;
  let fixture: ComponentFixture<EventPageComponent>;

  beforeEach(async () => {
    await TestBed.configureTestingModule({
      declarations: [EventPageComponent],
      schemas: [CUSTOM_ELEMENTS_SCHEMA],
    }).compileComponents();
  });

  beforeEach(() => {
    fixture = TestBed.createComponent(EventPageComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('event-page snapshot', () => {
    expect(fixture).toMatchSnapshot();
  });
});
