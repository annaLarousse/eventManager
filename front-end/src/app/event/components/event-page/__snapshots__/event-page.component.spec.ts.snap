// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventPageComponent event-page snapshot 1`] = `
<app-event-page
  __ngContext__={[Function LRootView]}
>
  <div
    class="event-page__header"
  >
    <img
      alt="Event Manager"
      class="logo"
      src="../../../../assets/logo.svg"
    />
  </div><div
    class="event-page__container"
  >
    <app-event-add />
    <mat-divider />
    <app-event-list />
  </div>
</app-event-page>
`;
